packages/expo-image-picker/src/ImagePicker.ts

```diff
@@ -11,6 +11,26 @@ import {
   VideoExportPreset,
 } from './ImagePicker.types';
 
+function validateOptions(options: ImagePickerOptions) {
+  const { aspect, quality, videoMaxDuration } = options;
+
+  if (aspect != null) {
+    const [x, y] = aspect;
+
+    if (x <= 0 || y <= 0) {
+      throw new CodedError('ERR_INVALID_ARGUMENT', 'Invalid aspect ratio values');
+    }
+  }
+
+  if (quality && (quality < 0 || quality > 1)) {
+    throw new CodedError('ERR_INVALID_ARGUMENT', 'Quality value must be between 0 and 1');
+  }
+
+  if (videoMaxDuration && videoMaxDuration < 0) {
+    throw new CodedError('ERR_INVALID_ARGUMENT', 'videoMaxDuration must be a non-negative number');
+  }
```

Comment:
```suggestion
  }
  return { aspect, quality, videoMaxDuration };
```

Reply:
I need to return whole `options` object, not only three params.